plugins/search-backend/src/service/router.ts

```diff
@@ -18,23 +18,24 @@ import express from 'express';
 import Router from 'express-promise-router';
 import { Logger } from 'winston';
 import { SearchQuery, SearchResultSet } from '@backstage/search-common';
+import { SearchEngine } from '@backstage/plugin-search-backend-node';
 
 type RouterOptions = {
+  engine?: SearchEngine;
```

Comment:
I made it optional because we are using [`createRouter`](https://github.com/backstage/backstage/blob/master/plugins/search-backend/src/service/standaloneServer.ts#L33) in the `startStandaloneServer` and not passing through engine there, but maybe we should?

Reply:
🤔  Yeah, I guess we're going to want to have the search endpoint return _something_ in standlone dev mode.  ...But, with no other processes running...  Not sure what the contents of the engine would be?  I guess we'd have to stub out some documents/indices too, then feed it to the engine, then provide the engine.